src/main/pages/che-6/portable-workspaces/factories_json_reference.adoc

```diff
@@ -197,13 +213,13 @@ Each event type has a set of actions that can be triggered. There is no ordering
 [width="100%",cols="34%,33%,33%",options="header",]
 |===
 |Action |Properties? |Description
-|`warnOnClose` |No |Opens a warning popup when the user closes the browser tab with a project that has uncommitted changes. Requires `project.parameters.keepVcs` to be `true`.
+|`warnOnClose` |No |Opens a warning pop-up window when the user closes the browser tab with a project that has uncommitted changes. Requires `project.parameters.keepVcs` to be `true`.
 |===
 
 [id="action-open-file"]
 == Action: Open File
 
-This action will open a file as a tab in the editor. You can provide this action multiple times to have multiple files open. The file property is a relative reference to a file in the project’s source tree. The `file` parameter is the relative path within the workspace to the file that should be opened by the editor. The `line` parameter is optional and can be used to move the editor cursor to a specific line when the file is opened. Note that projects are located in the workspaces `/projects` folder.
+This action opens a file as a tab in the editor. You can provide this action multiple times to have multiple files open. The file property is a relative reference to a file in the project source tree. The `file` parameter is the relative path within the workspace to the file that should be opened by the editor. The `line` parameter is optional and can be used to move the editor cursor to a specific line when the file is opened. Note that projects are located in the workspaces `/projects` directory.
```

Comment:
It looks like "workspaces `/projects` directory" is a typo which was originally meant to show that a workspace owns a `/projects` directory. Can that be rewritten to avoid the workspace showing possession? "Note that projects are located in the `/projects` directory of a workspace" (or something along those lines)?

Also, considering the use of the word "note", would this make sense as an admonition?

Reply:
Changed to "Projects are located in the `/projects` directory of a workspace." ( Got rid of "not that" because there was nothing noteworthy there.